src/TrkAnaTreeMaker_module.cc

```diff
@@ -264,19 +264,19 @@ namespace mu2e {
       art::Handle<CrvDigiCollection>                 _crvDigis;
       art::Handle<CrvStepCollection>                 _crvSteps;
       // CRV -- fhicl parameters
-      bool _crvhits, _crvpulses;
+      bool _fillcrvcoincs, _fillcrvpulses;
       double _crvPlaneY;  // needs to move to KinKalGeom FIXME
       // CRV (output)
-      std::vector<CrvHitInfoReco> _crvhit;
+      std::vector<CrvHitInfoReco> _crvcoincs;
       std::map<BranchIndex, std::vector<CrvHitInfoReco>> _allBestCrvs; // there can be more than one of these per track type
-      std::vector<CrvHitInfoMC> _crvhitmc;
+      std::vector<CrvHitInfoMC> _crvcoincsmc;
       std::map<BranchIndex, std::vector<CrvHitInfoMC>> _allBestCrvMCs;
       CrvSummaryReco _crvsummary;
       CrvSummaryMC   _crvsummarymc;
-      std::vector<CrvPlaneInfoMC> _crvhitmcplane;
-      std::vector<CrvPulseInfoReco> _crvpulseinfo;
-      std::vector<CrvWaveformInfo> _crvwaveforminfo;
-      std::vector<CrvHitInfoMC> _crvpulseinfomc;
+      std::vector<CrvPlaneInfoMC> _crvcoincsmcplane;
+      std::vector<CrvPulseInfoReco> _crvpulses;
+      std::vector<CrvWaveformInfo> _crvdigis;
```

Comment:
I'm afraid I'm now confused as to the difference between pulses, waveforms, and digis...

Reply:
I agree it's confusing, I had to ask Yuri. 

`digis` and `waveforms` are the same thing, they're ADU counts binned in 12.5 ns intervals. `pulses` are reconstructed distributions of `digis` (fitted and integrated).

Reply:
Ah, OK let's move to just using ```digis`` then and try to remove ```waveform``` everywhere we can